src/main/java/com/softserveinc/ita/utils/runners/TestRunner.java

```diff
@@ -1,16 +1,20 @@
 package com.softserveinc.ita.utils.runners;
 
+import com.softserveinc.ita.pageobjects.CategoriesPage;
 import com.softserveinc.ita.pageobjects.HomePage;
+import com.softserveinc.ita.pageobjects.ProductPage;
 import org.testng.annotations.BeforeMethod;
 
 import static com.codeborne.selenide.Selenide.open;
 
 public abstract class TestRunner {
 
+    protected CategoriesPage categoriesPage = new CategoriesPage();
```

Comment:
init pages only in tests or get them through return statements in methods

delete pages from here, you can init homepage in homepagetest class